src/main/java/objectsBasicsMandatoryPracticalTasks/Author.java

```diff
@@ -0,0 +1,16 @@
+package objectsBasicsMandatoryPracticalTasks;
+
+class Author {
+    private String firstName, lastName;
+
+    Author(String firstName, String lastName) {
+        this.firstName = firstName;
+        this.lastName = lastName;
+    }
+
+    void printAuthor(Author author) {
```

Comment:
This method is not static, so no need to pass author into arguments.
You can access author details directly as you did.